release_index.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+// ReleaseIndex for accessing Release Index metadata
+type ReleaseIndex struct {
+	Releases []Index `json:"releases"`
+}
+
+// Index - details for single release
+type Index struct {
+	Metadata string `json:"metadata"`
+}
+
+// ReleaseURL gets path for latest release.json available
+func ReleaseURL(releaseIndexURL string) (string, error) {
+	var relIndex ReleaseIndex
+	resp, err := http.Get(releaseIndexURL)
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
+
+	decoder := json.NewDecoder(resp.Body)
+	if err := decoder.Decode(&relIndex); err != nil {
+		return "", err
+	}
+
+	return relIndex.Releases[len(relIndex.Releases)-1].Metadata, nil
```

Comment:
It would be better to have a check that `len(relIndex.Releases) > 0`, this may panic.

Reply:
Indeed, nice catch. fixed!